west.yml

```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: edf807fd47cd6c3ab71c020363888fe73f2cc776
+      revision: pull/1328/head
```

Comment:
SHA can be updated: `06088342cb531ee470c7f64c8c7c0e10d75de8fc`

Reply:
@rlubos - add the SHA. 

Thanks and regards
Vivek